### ISSUE TITLE
FT as front-end for existing database

### ISSUE THREAD

**Issue by artelogic-com** [formtools/core#354]
Is it possible to use FT with an existing DB schema instead of creating one for the submissions?

**Comment by benkeen** [comment]
Hey @artelogic-com, sorry for not responding sooner, I've been out of the country. 

> Is it possible to use FT with an existing DB schema instead of creating one for the submissions?

I'm afraid not. Neat idea, for sure - and it'd be definitely doable (to some degree) but I'm afraid that feature isn't offered at the moment. 

All the best -

Ben

**Comment by WisTex** [comment]
It would be nice if you added this feature.

**Comment by benkeen** [comment]
Agreed, it'd be nice. I'll re-open and leave as an open feature request. But yeah - no plans to do this any time soon, sorry!
